app/config/image_variations.yml

```diff
@@ -39,3 +39,9 @@ ezpublish:
                     filters:
                         - { name: geometry/scaledownonly, params: [420, 370] }
                         - { name: geometry/crop, params: [420, 270, 0, 0] }
+
+                card_view:
+                    reference: null
+                    filters:
+                        - {name: geometry/scalewidth, params: [255]}
+                        - {name: geometry/crop, params: [255, 175, 0, 0]}
```

Comment:
Is the crop necessary?
I am wondering about images uploaded with a different aspect ratio.

Reply:
the same filter is used for product card image, but with different dimentions